src/verifyEmail.js

```diff
@@ -0,0 +1,27 @@
+function verifyEmail(appId, serverURL) {
```

Comment:
To keep with the current pattern, maybe we should have an `class EmailVerificationController extends PromiseRouter {}`.